src/resources/games-toolkit.md

```diff
@@ -1,241 +1,382 @@
-[check your region's eligibility][]).
+you can leverage up to $900 in offers when you integrate Google services,
+such as [Cloud, Firebase][], and [Ads][], into your game.
 
-[Ads]: https://ads.google.com/intl/en_us/home/flutter/#!/
-[check your region's eligibility]: https://www.google.com/intl/en/ads/coupons/terms/flutter/
-[Cloud, Firebase]: https://cloud.google.com/free
-[Games]: {{site.main-url}}/games
+{{ site.alert.important }}
+**Terms and conditions apply.**
+You must connect your Firebase and GCP accounts to use credits for
+Firebase services and verify your business email during sign up to earn
+an additional $100 on top of the normal $300 credit.
+For the Ads offer, [check your region's eligibility][].
+{{ site.alert.end }}
 
 ## Get started
 
 Are you ready? To get started:
 
 1. [Watch the Quick Start video][game-video]
-1. [Clone the games template repo][game-template], and review the [template readme][game-template-readme]
-1. [Join the Flutter Games community on Discord][game-discord]
-   (use the [direct link][direct-discord] if you already
+1. [Clone the games repo][game-repo].
+1. Review the `README` file for the first type of game you want to create.
+
+   * [basic game][basic-template-readme]
+   * [card game][card-template-readme]
+   * [runner game][runner-template-readme]
+
+1. [Join the Flame community on Discord][game-discord]
+   (use the [direct link][discord-direct] if you already
    have a Discord account).
 1. Set up accounts on AdMob, Firebase, and Cloud, as needed
 1. Write your game!
 1. Deploy to both the Google and Apple stores
 
-[direct-discord]: https://discord.com/login?redirect_to=%2Fchannels%2F420324994703163402%2F964110538986651658
-[game-discord]: https://discord.com/invite/WY5NwwjBQz
-[game-template]: {{site.github}}/flutter/samples/tree/main/game_template
-[game-video]: {{site.youtube-site}}/watch?v=zGgeBNiRy-8&t=3s
-[game-template-readme]: {{site.github}}/flutter/samples/tree/main/game_template#readme
+## Advanced game development
 
+Perhaps you want to write a hobbyist video game?
+The game template still provides value for
+the advanced interface and structure of such games.
 
-## Video games
+For hobbyist games, check out the
+[Flame package][flame-package] and join the
+[Flame community on Discord][discord-flame].
 
-Perhaps you want to write a video-style game?
-The game template is still valuable for
-the high-level structure of such games.
+## Example games
 
-For video games, you might want to look into the
-[Flame package][flame-package],
-join the [Flame community on Discord][flame-discord],
-and play with the [Flame demo app on DartPad][flame-demo].
-You might also check out the [Flutter + Flame codelab][].
+For Google I/O 2022, both the Flutter team and Very Good Ventures created new
+games.
 
-[Flutter + Flame codelab]: {{site.codelabs}}/codelabs/flutter-flame-game
+* VGV created the [I/O Pinball game][pinball-game] using the Flame engine.
```

Comment:
Do we know, for sure, that this is up to date?

Reply:
As far as we know, yes. Zoey didn't strike it.